cmd/promtool/tsdb.go

```diff
@@ -624,3 +625,11 @@ func checkErr(err error) int {
 	}
 	return 0
 }
+
+func readOpemmetrics(path string, outputDir string, DefaultBlockDuration time.Duration) (err error) {
```

Comment:
Since this is an argument, it's not a "default" (and generally capitalise variables only when it's a global to avoid confusions).

```suggestion
func readOpemmetrics(path string, outputDir string, blockDuration time.Duration) (err error) {
```